buildSrc/src/main/groovy/Dependencies.groovy

```diff
@@ -16,6 +16,8 @@ class Dependencies {
   static def springBootStarterWeb = { it=Versions.springBoot -> "org.springframework.boot:spring-boot-starter-web:${it}" }
   static def springDataSolr = { it=Versions.springData -> "org.springframework.data:spring-data-solr:${it}" }
   static def transformAPI = { it=Versions.transformAPI -> "com.connexta.transformation:transformation-api-rest-spring-stubs:${it}" }
+  static def swagger = { it=Versions.swagger -> "io.springfox:springfox-swagger2:${it}" }
```

Comment:
❗️ It looks like Springfox isn't supporting OpenAPI 3.0 yet and it's been a year since they have cut a release: https://github.com/springfox/springfox/releases . I'm not sure if we even want to merge this if it won't be the same API version as our other repos.

Reply:
Springfox does not support OAS 3.0.Springfox swagger does not ingest an OpenAPI file, it looks at the Spring MVC annotations on the controller to generate information.

 It looks like the *right* way to run swagger UI is as a separate service. I hope that is something the platform team eventually provides.